docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -385,6 +385,12 @@ that stores spooled data in a specified location, such as {ref}`AWS S3
 <fte-exchange-azure-blob>`, {ref}`Google Cloud Storage <fte-exchange-gcs>`,
 or {ref}`HDFS <fte-exchange-hdfs>`.
 
+:::{note}
```

Comment:
Not sure it should be a note. Just a normal sentence is probably fine. Also it should be in the section about S3 not in the overview here.

Reply:
Also .. dont use "we"